Documentation/raspi-spi.md

```diff
@@ -0,0 +1,135 @@
+# Enabling SPI on Raspberry Pi
+
+In most of the cases, it is easy and straight forward to enable SPI for your Raspberry Pi. The basic case can be [found here](https://www.raspberrypi-spy.co.uk/2014/08/enabling-the-spi-interface-on-the-raspberry-pi/).
+
+This page will explain how to setup any SPI. Please refer to the [Raspberry Pi documentation](https://www.raspberrypi.org/documentation/hardware/raspberrypi/spi/) to understand the different SPI available. You should be aware as well that for Raspberry Pi4, some of the configurations are different than for the other version especially for SPI3, 4 and 5. 
+
+## Basic Hardware SPI usage
+
+The most simplest code you can build to use SPI is the following:
+
+```csharp
+using System;
+using System.Device.Spi;
+
+namespace TestTest
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            SpiDevice spi = SpiDevice.Create(new SpiConnectionSettings(0));
+            spi.WriteByte(0x42);
+            var incoming = spi.ReadByte();
+        }
+    }
+}
+```
+
+This will open SPI0, with the default Chip Select. It will then write a byte to the MOSI pin and read 1 byte from the MISO pin. 
+
+If you get something like this, it means you need to check the next sections to activate your SPI0:
+
+```
+Unhandled exception. System.IO.IOException: Error 2. Can not open SPI device file '/dev/spidev0.0'.
+   at System.Device.Spi.UnixSpiDevice.Initialize()
+   at System.Device.Spi.UnixSpiDevice.WriteByte(Byte value)
+   at SpiTest.Program.Main(String[] args) in C:\tmp\TestTest\SpiTest\Program.cs:line 11
+Aborted
+```
+
+
+## Enabling SPI0 without Hardware Chip Select
+
+In very short, this is the line you'll need to add into the `/boot/config.txt` file:
+
+```bash
+sudo nano /boot/config.txt
+```
+
+Add the line: 
+
+```
+dtparam=spi=on
+```
+
+Save the file with `ctrl + x` then `Y` then `enter`
+
+Then reboot:
+
+```bash
+sudo reboot
+```
+
+This will enable SP0 where those are the pins which will be seleted, only Software Chip Select is activated with the default pins:
```

Comment:
```suggestion
This will enable SPI0 where those are the pins which will be selevted, only Software Chip Select is activated with the default pins:
```

Reply:
"selected" is still spelled incorrectly ;-)